contributors/devel/sig-release/cherry-picks.md

```diff
@@ -90,9 +90,11 @@ patch release branches.
     release you want to cherry pick to. Cherry picks should be applied to all
     [active](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#detailed-release-history-for-active-branches)
     release branches where the fix is applicable.
-  
-  - When asked for your github password, provide the created github token 
-    rather than your actual github password. 
+
+  - Set the value of the environment variable `GITHUB_TOKEN` to your personal
```

Comment:
nit: I'd suggest to add the option of adding environment variables as an option under the interactive one. First, reader likely started the command already. So interactive option is more important. Second, setting PAT in environment variable is not super secure.

Reply:
Makes sense!